microhapulator/data/template.html

```diff
@@ -25,14 +30,34 @@
             table {
                 margin: 12pt auto;
                 width: 100%;
+                display: block;
+                max-width: -moz-fit-content;
+                max-width: fit-content;
```

Comment:
Another possible issue with cross-browser compatibility: the `-moz` prefix makes me wonder whether this setting is only honored by Mozilla/Firefox. With Safari on Mac, I'm seeing some inconsistencies with table width and alignment. Before I provide a laundry list of observations, maybe you could check for consistency across browsers. Once you've confirmed that things are behaving as intended, I'll revisit that question.

Reply:
Turns out the table width and alignment issue is caused by trying to make the table width 100% and make it scrollable. The solution is to set the table width to 100% and then create a separate scrollable wrapper div.  The tables look much better in my testing, but let me know if you see anything that still looks off. I deleted the -moz-fit-content because it wasn't actually needed.

Reply:
Looks good!